install.sh

```diff
@@ -226,7 +226,7 @@ main() {
         test -r .bash_profile && echo 'PATH=~/bin:$PATH' >> .bash_profile || \
             echo 'PATH=~/bin:$PATH' >> .profile
     fi
-    if test -f .zshrc; then
+    if [ -f .zshrc || $SHELL == "/bin/zsh" ]; then
```

Comment:
```suggestion
    if [ -f .zshrc ] || [[ "$SHELL" == "/bin/zsh" ]]; then
```
after playing at the command prompt a bit, I think this is what you need